collections/poker-club.csv

```diff
@@ -0,0 +1,201 @@
+name
+2♠3♠

```

Comment:
bad file. File must will be example
2♠3♠,243535423524363456345654364536345643

please read readme.md

Reply:
I thought no ids for new collection

Reply:
has anything changed already? At least an ID was needed before.